sdks/go/pkg/beam/core/runtime/exec/combine.go

```diff
@@ -499,3 +499,30 @@ func (n *ExtractOutput) ProcessElement(ctx context.Context, value *FullValue, va
 	}
 	return n.Out.ProcessElement(n.Combine.ctx, &FullValue{Windows: value.Windows, Elm: value.Elm, Elm2: out, Timestamp: value.Timestamp})
 }
+
+// ConvertToAccumulators is an executor for converting an input value to an accumulator value.
+type ConvertToAccumulators struct {
+	*Combine
+}
+
+func (n *ConvertToAccumulators) String() string {
```

Comment:
Please add a test like Combine/LiftedCombine: https://github.com/apache/beam/blob/c8e200b157b43ad471c9480f8f617d9fbef33070/sdks/go/pkg/beam/core/runtime/exec/combine_test.go#L84

Reply:
Talked with Robert and we came up with TestConvertToAccumulators.